<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicsStreamingClass</class>
 <widget class="QMainWindow" name="MusicsStreamingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicsStreaming</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnDownload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MP3 Download</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dap_premium.png</normaloff>:/images/dap_premium.png</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="downloadProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMusicStreaming">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ico_tutorial.png</normaloff>:/images/ico_tutorial.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtServer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>DM01FTD0007</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SongID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/Satellite.png</normaloff>:/images/Satellite.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="mp3List">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionSet_Server"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <action name="actionSet_Server">
   <property name="text">
    <string>Set Server</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mediaplayer.qrc"/>
 </resources>
 <connections/>
</ui>
